CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v12.0.0 (upcoming)
+
+* Add `django v1.8` support.
+
```

Comment:
As far as I can see this is backwards compatible, so should probably be `11.1.0`